Firestore/core/src/firebase/firestore/core/sync_engine.h

```diff
@@ -41,21 +41,7 @@ namespace firebase {
 namespace firestore {
 namespace core {
 
-/**
- * Interface implemented by `EventManager` to handle notifications from
- * `SyncEngine`.
- */
-class SyncEngineCallback {
```

Comment:
Did we have it defined twice (here and in `sync_engine_callback.h`)?

Reply:
Yes.